apps/web/src/app/globals.css

```
@@ -434,6 +434,15 @@
   padding-right: env(safe-area-inset-right, 0);
 }
 
+/* Hide native scrollbar — cross-browser utility for horizontal scroll containers */
+@utility scrollbar-hide {
+  scrollbar-width: none;            /* Firefox */
+  -ms-overflow-style: none;         /* IE / Edge */
+  &::-webkit-scrollbar {
```

Stylelint (17.6.0)

[error] Unexpected missing scoping root (nesting-selector-no-missing-scoping-root)

(nesting-selector-no-missing-scoping-root)